### DISCUSSION TITLE
Sketch to 3D model in stable diffusion???

### DISCUSSION THREAD

**Discussion by kukovinetsyevhenii** [lllyasviel/Fooocus#3468] [Q&A] [answered]
Can we create an efficient and stable diffusion system for converting sketches into 3D models?
If not possible with Fooocus, what is ideal choice for this?

**Comment by mashb1t** [comment, marked as answer]
Fooocus does create images and not 3d models. Appropriate open source tools tool for this use case would be https://github.com/threestudio-project/threestudio (most likely outdated), https://github.com/openai/shap-e (also most likely outdated) and https://github.com/ashawkey/stable-dreamfusion (same same), but paid tools also exist, which seem to get more attention.

**Comment by IPv6** [comment]
MeshAnything V2 seems to be promising too

**Reply by mashb1t** [reply]
=> https://github.com/buaacyw/MeshAnything

**Reply by IPv6** [reply]
That` s V1, V2 => https://github.com/buaacyw/MeshAnythingV2
Best part topology seems to be really usable this time
